.github/workflows/pages.yml

```diff
@@ -10,8 +10,6 @@ on:
   # Runs on pushes targeting the default branch
   push:
     branches: ["master"]
-  schedule:
-    - cron: '0 0 * * *'
```

Comment:
this is because the [plugins page](https://www.chaijs.com/plugins/) gets dynamically generated.

Reply:
Ah! I'll put this back.